### ISSUE TITLE
Can't use setUnreachedBarColor 

### ISSUE THREAD

**Issue by TCleard** [daimajia/NumberProgressBar#25]
Hello, there.

First of all, awesome library, it works great ;)

But I have a problem setting the unreached bar color, it doesn't work exactly as expected :

``` java
private void initColors() {
    _progress.setReachedBarColor(getResources().getColor(R.color.blue));
    _progress.setUnreachedBarColor(getResources().getColor(R.color.white));
}
```

Here, the reached color is good, but the unreached color isn't white, but blue (as the reached color).

And if we look at your code, we can see a little mistake :

``` java
public void setUnreachedBarColor(int BarColor) {
    this.mUnreachedBarColor = BarColor;
    mUnreachedBarPaint.setColor(mReachedBarColor);
    invalidate();
}
```

In the setUnreachedBarColor function, you update mUnreachedBarColor, but you don't use it to color the unreached bar, you use mReachedBarColor.

Hope you'll fix this soon ;)

Android Studio 1.4 
Grade : compile 'com.daimajia.numberprogressbar:library:1.2@aar'

**Comment by TCleard** [comment]
By the way, setting the color from xml is working just fine.
custom:progress_reached_color="@color/blue"
custom:progress_unreached_color="@color/white"

**Comment by ahmed-basyouni** [comment]
first i would like to say it's an awesome lib 
In case anyone still got this issue 

go to lib source code and change 

public void setUnreachedBarColor(int barColor) {
        this.mUnreachedBarColor = barColor;
        mUnreachedBarPaint.setColor(mReachedBarColor);
        invalidate();
    }

to 

public void setUnreachedBarColor(int barColor) {
        this.mUnreachedBarColor = barColor;
        mUnreachedBarPaint.setColor(mUnreachedBarColor);
        invalidate();
    }

**Comment by Dona278** [comment]
I've forked this repository and published with same groupId and artifact of the original author but i've added some features and bug fix.
Try it https://github.com/Dona278/NumberProgressBar

**Comment by daimajia** [comment]
Fixed in version 1.3. Thanks your feedback.
